src/main/scala/com/campudus/tableaux/controller/SystemController.scala

```diff
@@ -332,4 +348,91 @@ class SystemController(
       _ <- serviceModel.delete(serviceId)
     } yield EmptyObject()
   }
+
+  def retrieveCellAnnotationConfigs()(implicit user: TableauxUser): Future[DomainObject] = {
+    logger.info(s"retrieveCellAnnotationConfigs")
+    for {
+      annotations <- cellAnnotationConfigModel.retrieveAll()
+    } yield {
+      val json = Json.obj("annotations" -> annotations.map(_.getJson))
+
+      PlainDomainObject(json)
+    }
+  }
+
+  def retrieveCellAnnotationConfig(annotationName: String)(implicit
+  user: TableauxUser): Future[CellAnnotationConfig] = {
+    logger.info(s"retrieveCellAnnotationConfig $annotationName")
+
+    for {
+      _ <- roleModel.checkAuthorization(ViewCellAnnotationConfig)
+      annotation <- cellAnnotationConfigModel.retrieve(annotationName)
+    } yield annotation
+  }
+
+  def deleteCellAnnotationConfig(annotationName: String)(implicit user: TableauxUser): Future[DomainObject] = {
+    logger.info(s"deleteCellAnnotationConfig $annotationName")
+
+    for {
+      _ <- roleModel.checkAuthorization(DeleteCellAnnotationConfig)
+      _ <- cellAnnotationConfigModel.delete(annotationName)
+    } yield EmptyObject()
+  }
+
+  def updateCellAnnotationConfig(
+      name: String,
+      priority: Option[Int],
+      fgColor: Option[String],
+      bgColor: Option[String],
+      displayName: Option[MultiLanguageValue[String]],
+      isMultilang: Option[Boolean],
+      isDashboard: Option[Boolean]
+  )(implicit user: TableauxUser): Future[DomainObject] = {
+
+    checkArguments(
+      notNull(name, "name"),
+      isDefined(
+        Seq(priority, fgColor, bgColor, displayName, isMultilang, isDashboard),
+        "priority, fgColor, bgColor, displayName, isMultilang, isDashboard"
+      )
+    )
+
+    logger.info(
+      s"updateCellAnnotationConfig $name $priority $fgColor $bgColor $displayName $isMultilang $isDashboard"
+    )
+
+    for {
+      _ <- roleModel.checkAuthorization(EditCellAnnotationConfig)
```

Comment:
Da müssten mir mit @Zwergal klären, ob wir das so fein granular haben wollen, oder für die Zukunft brauchen...
Macht schon Sinn, falls die Kunden das später evtl. selbst definieren können sollen. Ich glaube das wird aber eher initial beim Projektsetup gemacht und für die Wartung und Config der GRUD könnten wir das auch zum globalen Recht `EditSystem` hinzufügen.

Reply:
Wie @zingmane schreibt würde erstmal reichen wenn es beim `EditSystem` dabei ist. Später könnte es aber schon interessant sein, das einzeln zu haben, wenn wir Benutzern das selbst machen lassen.
Von dem her können wir es gerne so lassen und wir nehmen das Recht mit in unsere Admin/dev-Rollen mit auf. dann können es auch erstmal nur wir aber wir sind auch für die Zukunft gewappnet.

Reply:
Ok, muss dazu code-technisch noch etwas erweitert werden?
Bisher habe ich nur die `role-permissions-example.json` und `role-permissions-test.json` erweitert.